### DISCUSSION TITLE
Compatibility issue with @Changelog

### DISCUSSION THREAD

**Discussion by Liveitabhi14** [flamingock/mongock#42] [Q&A] [answered]
Are there any compatibility issues expected on using @Changelog with any Mongo version? (mongo 4.2.19 in my case)

I am trying to connect my spring boot app to MongoDB on a 4.2.19 mongo server. Mongock execution fails throwing "read time out exception.. ABORTED MIGRATION" while the app tries to come up.

The app works totally fine when I connect to a DB on 4.2.15 mongo server (with no exceptions for Mongock)

I am using Mongock 4

**Comment by dieppa** [comment, marked as answer]
Hello, there is no identified issue in that contact as far as we are aware. 

Are you sure this is produced by Mongock?
